src/features/specialist/components/SpecialistReviewList.tsx

```
@@ -0,0 +1,57 @@
+
+export const ReviewsList = ({ reviews }: Props) => {    
+  
+  if (reviews.length === 0) {
+    return (
+      <div className="text-center text-lg text-shark py-10">
+        Ще немає відгуків. Будьте першим, хто залишить відгук!
+      </div>
+    );
+  }
+
+  const {} = reviews[0];
```

ESLint

[error] Unexpected empty object pattern.

(no-empty-pattern)